build/Dockerfile.ci

```diff
@@ -33,6 +33,18 @@ RUN git clone --single-branch --branch release-4.6 https://github.com/openshift/
 WORKDIR ovn-kubernetes/go-controller/
 RUN make windows
 
+# Build promu utility tool
+WORKDIR /build/
+RUN git clone --single-branch --branch v0.6.1 https://github.com/prometheus/promu
+WORKDIR /build/promu/
+RUN make build
+
+# Build windows_exporter
+WORKDIR /build/
+RUN git clone --single-branch --branch v0.13.0 https://github.com/prometheus-community/windows_exporter
```

Comment:
Do you want to pin to a commit in case 0.13.0 gets updated with some changes?

Reply:
Yes, sounds good, thank you.

Reply:
On second thought, since 0.13.0 is not the latest release, leaving this as it is seems fine to me. Let me know if you think otherwise.

Reply:
`v0.13.0` is a tag which typically should not get updated. I also see that `v0.14.0` has been released a month ago. Please move to that if it does not break things.